tools/rules/trailing_whitespace

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/bin/bash
 #
 # Checks whether source files contain trailing script.
```

Comment:
Oops. I suppose, I originally meant to type "Checks whether **any** source files contain trailing **white space**".

Reply:
Fixed :)